pkg/common/log/reopen_test.go

```diff
@@ -0,0 +1,75 @@
+package log
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReopenOnSignal(t *testing.T) {
+	const (
+		_tempDir         = "spirelogrotatetest"
+		_testLogFileName = "test.log"
+		_rotatedSuffix   = "rotated"
+		_firstMsg        = "a message"
+		_secondMsg       = "another message"
+	)
+	dir, err := os.MkdirTemp("", _tempDir)
```

Comment:
We have a utility called `spiretest.TempDir()` that simplifies the creation of temporary directories

Reply:
Ack.